app/helpers/index.js

```diff
@@ -0,0 +1,20 @@
+const users = require('../models').users,
+  errors = require('../errors'),
+  isAlphanumeric = require('is-alphanumeric');
+
+exports.validateEmailPassword = (email, pwd) => {
+  const VALID_AR = /^\w+([\.-]?\w+)*@\wolox.com.ar/,
+    VALID_CO = /^\w+([\.-]?\w+)*@\wolox.co/,
+    VALID_CL = /^\w+([\.-]?\w+)*@\wolox.cl/;
+  return new Promise(function(resolve, reject) {
+    if (VALID_AR.test(email) || VALID_CO.test(email) || VALID_CL.test(email)) {
+      if (isAlphanumeric(pwd) && pwd.length >= 8) {
```

Comment:
theres no need of using a dependecy for validating alphanumeric, you can use another regexp